### ISSUE TITLE
System.NullReferenceException after start

### ISSUE THREAD

**Issue by denbkh** [lostindark/DriverStoreExplorer#160]
Hi, when I try to run the application I get the System.NullReferenceException error.
App version is 0.11.64.
Here is the data from the log file:
```
2021-04-03 09:32:44Z [Information]: Rapr started
2021-04-03 09:32:44Z [Information]: Scanning driver store...
2021-04-03 09:32:46Z [Error]: System.NullReferenceException: Object reference not set to an instance of an object.
   at Rapr.DSEForm.<>c.<SetupListViewColumns>b__11_8(Object condition) in C:\projects\driverstoreexplorer\Rapr\DSEForm.cs:line 174
   at BrightIdeasSoftware.OLVColumn.ValueToString(Object value)
   at BrightIdeasSoftware.ObjectListView.MakeSubItem(Object rowObject, OLVColumn column)
   at BrightIdeasSoftware.ObjectListView.FillInValues(OLVListItem lvi, Object rowObject)
   at BrightIdeasSoftware.ObjectListView.BuildList(Boolean shouldPreserveState)
   at BrightIdeasSoftware.ObjectListView.SetObjects(IEnumerable collection, Boolean preserveState)
   at Rapr.DSEForm.<PopulateUIWithDriverStoreEntries>d__16.MoveNext() in C:\projects\driverstoreexplorer\Rapr\DSEForm.cs:line 249
```

**Comment by denbkh** [comment]
Workaround: disable last 2 columns here:
![2021-04-03_12-51-29](https://user-images.githubusercontent.com/4642384/113475041-feb38c00-947b-11eb-9d98-91a36aaa203f.png)

**Comment by lostindark** [comment]
Thanks for reporting. Fixed in 282362b51773293cfda3bcb5e8d220339c961a9f.
